CTags.tmbundle/Support/update_tags.php

```diff
@@ -2,8 +2,9 @@
 
 require_once __DIR__ . DIRECTORY_SEPARATOR . 'bootstrap.php';
 
+
 $cmd = implode(' ', array(
-	escapeshellcmd(BUNDLE_SUPPORT . 'bin' . DS . 'ctags'),
+	escapeshellcmd(escapeshellarg(BUNDLE_SUPPORT) . 'bin' . DS . 'ctags'),
```

Comment:
`escapeshellcmd` does not seem to deal with path escaping for paths that contain a space like `Library/Application Support/Textmate` `escapeshellarg` creates a compatible path with the space escaped like `Library/Application\ Support/Textmate`. I don't know PHP well enough I guess to know why that is :(